<a href="https://colab.research.google.com/github/mgmills93/AEM_Spring_2019/blob/master/SecureCommunities_FinalProject/Data_Analysis/FinalProject_Nationality_ANALYSIS.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

###Question 2: What nationalities are being deported the most?
---
I will arrange my country_sums.csv set to see what nationality was the most represented in deported individuals through Secure Communities from 2015-2017.

In [0]:
import pandas as pd

country_sums = pd.read_csv('https://raw.githubusercontent.com/mgmills93/AEM_Spring_2019/master/SecureCommunities_FinalProject/Datasets/country_sums.csv')

def top_5(country_sums):
  country_sort = country_sums.sort_values(by=['Arrest Date'], ascending=False)
  country_sort = country_sort.reset_index(drop=True)
  
  return country_sort
country_sort = top_5(country_sums)

In [0]:
def clean(country_sort):
  country_sort.rename(columns={'Arrest Date':'Arrest_Number'}, inplace=True)
  final_sort = country_sort[['Citizenship Country', 'Arrest_Number']]
  
  return final_sort

###Pipeline


In [26]:
#sort country_sums by largest number of arrests
top_5(country_sums)
#clean sorted data
clean(country_sort)

Citizenship Country  Arrest_Number
0                        MEXICO         113697
1                      HONDURAS           9086
2                     GUATEMALA           8610
3                   EL SALVADOR           6859
4            DOMINICAN REPUBLIC           1800
5                       JAMAICA            871
6                      COLOMBIA            829
7                         HAITI            545
8                       ECUADOR            508
9                     NICARAGUA            479
10                       BRAZIL            431
11                       CANADA            358
12                         PERU            267
13                        INDIA            255
14   CHINA, PEOPLES REPUBLIC OF            221
15                  PHILIPPINES            191
16               UNITED KINGDOM            186
17                      NIGERIA            156
18                      ROMANIA            143
19                  SOUTH KOREA            119
20                      BAHAMAS            111
21                 SAUDI ARABIA            111
22          TRINIDAD AND TOBAGO            107
23                        KENYA            106
24                   COSTA RICA            103
25                       POLAND             98
26                        CHILE             98
27                       GUYANA             97
28                      UNKNOWN             95
29                    VENEZUELA             93
..                          ...            ...
159    TURKS AND CAICOS ISLANDS              2
160                TURKMENISTAN              2
161                     BAHRAIN              2
162                        USSR              2
163                       QATAR              2
164                     BERMUDA              2
165                     FINLAND              2
166                     BURUNDI              2
167                     NAMIBIA              2
168                     DENMARK              2
169                   STATELESS              2
170            PAPUA NEW GUINEA              1
171                       ARUBA              1
172                    ANGUILLA              1
173                        OMAN              1
174                       MALTA              1
175                  MOZAMBIQUE              1
176                     LESOTHO              1
177                   MAURITIUS              1
178        UNITED ARAB EMIRATES              1
179                     COMOROS              1
180                 PUERTO RICO              1
181                    BOTSWANA              1
182                        GUAM              1
183                       SAMOA              1
184                       SYRIA              1
185       SERBIA AND MONTENEGRO              1
186                   SWAZILAND              1
187                  SEYCHELLES              1
188                      CYPRUS              1

[189 rows x 2 columns]

###Analysis
---
The top 5 countries with the most nationals deported from the U.S. from 2015-2017 were:
Mexico - 113697  
Honduras -	9086  
Guatemala -	8610  
El Salvador -	6859  
Dominican Republic - 1800  
JAMAICA	- 871  

These numbers are generally congruent with recent trends in asylum-seekers along the U.S.-Mexico border. This would suggest that many seeking asylum are crossing the border illegally, and these numbers represent those asylum-seeking individuals who commit crimes while in the U.S.